charts/substra-backend/templates/_helpers.tpl

```diff
@@ -194,4 +194,11 @@ The hostname we should connect to (external is defined, otherwise integrated)
     {{- else }}
         {{- template "postgresql.primary.fullname" .Subcharts.postgresql }}.{{ .Release.Namespace }}
     {{- end }}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+# charts/<your_chart>/templates/_helpers.tpl
```

Comment:
What is this line for?

Reply:
Sorry, my bad it should have been removed